packages/augur-ui/src/modules/events/actions/add-update-transaction.ts

```diff
@@ -103,6 +107,9 @@ export const addUpdateTransaction = (txStatus: Events.TXStatus) => async (
       methodCall !== CANCELORDER &&
       methodCall !== PUBLICFILLORDER
     ) {
+      if (methodCall === CREATEAUGURWALLET) {
```

Comment:
I don't think this is needed if the create augur wallet transaction is tracked in pending queue. see comment above.